hs_core/views/utils.py

```diff
@@ -293,13 +293,8 @@ def run_ssh_command(host, uname, pwd, exec_cmd):
     stdout = session.makefile('rb', -1)
     stdin.write("{cmd}\n".format(cmd=pwd))
     stdin.flush()
-    logger = logging.getLogger(__name__)
     output = stdout.readlines()
-    if output:
-        logger.debug(output)
-        return '.'.join(output)
-    else:
-        return ''
+    return output
```

Comment:
Why did you change this?  I think it makes sense to have debug logging in.  Also, I'm not sure the raw output from readlines is better to pass up than the string that came before.  I think it also probably makes sense to do the error checking in here instead of everywhere this method is used.

Reply:
The reason I removed logging statement is that a lot of informational output is written to the log. I have updated the user creation script to remove those echo statements so only command outputs will be captured. Will add the logging statement back in. No need to define ```logger``` variable inside the method, though, since the variable is already defined on the top of the module.